debug failed test cases (#146)
OctoMind-dev/mintlify-docs: debugtopus.mdx

Click on `run locally` to get a shell script snippet which you can use to install and start Debugtopus.

<Frame caption="screenshot 02/2024">
  <img src="images/test-result-failure.png" alt="screenshot 02/2024" />
</Frame>

Click on "copy to clipboard".

<Frame caption="screenshot 02/2024">
  <img src="images/screenshot-run-locally.png" alt="screenshot 02/2024" />
</Frame>

## 2. Start test locally

Open a terminal window and paste the shell script from your clipboard.

<Frame caption="screenshot 02/2024">
  <img
    src="images/Terminal-run-locally-npx-command-pasted.png"
    alt="screenshot 02/2024"
  />
</Frame>

> Now change the **url** with the url of your locally running application. In the example screenshot below

Reviewer: ```suggestion
Now change the **url** with the url of your locally running application. In the example screenshot below,
```

the placeholder was replaced with _localhost:3000_. You might also have to include the route to the starting point.

> For example, if your testcase assumes to run from the sign-in page you might have to add the route to the sign-in page

Reviewer: ```suggestion
For example, if your test case assumes to run from the sign-in page you might have to add the route to the sign-in page
```

## 4. Run test locally

> Click the run button which is displayed on the right side of the test case to run your test.

Reviewer: ```suggestion
Click the `run` button which is displayed on the right side of the test case to run your test.
```
